lib/livebook/runtime.ex

```diff
@@ -1101,4 +1101,10 @@ defprotocol Livebook.Runtime do
   """
   @spec unregister_clients(t(), list(client_id())) :: :ok
   def unregister_clients(runtime, client_ids)
+
+  @doc """
+  Fetches the running Proxy Handler's pid from runtime.
```

Comment:
We should document the messaging here, but we can do it once we move the proxy into Livebook.

```suggestion
  Fetches the running Proxy Handler's pid from runtime.

  TODO: document the communication here.
```